src/document.c

```diff
@@ -1273,7 +1273,6 @@ void document_show_tab(GeanyDocument *doc)
 		document_get_notebook_page(doc));
 }
 
-
```

Comment:
don't remove this line (it's part of the style -- and in any case, don't introduce style-only changes in the middle of a commit)